lib/ansible/cli/doc.py

```diff
@@ -312,10 +312,10 @@ class DocCLI(CLI, RoleMixin):
     _RULER = re.compile(r"\bHORIZONTALLINE\b")
 
     # rst specific
-    _REFTAG = re.compile(r":ref:")
-    _TERM = re.compile(r":term:")
-    _NOTES = re.compile(r".. note:")
-    _SEEALSO = re.compile(r"^\s*.. seealso:.*$", re.MULTILINE)
+    _RST_ANCHOR = re.compile(r"^\s*.. \w+?::.*$", re.MULTILINE)
+    _RST_REFTAG = re.compile(r":\w+?:`")
+    _RST_NOTES = re.compile(r".. notes::")  # subset of _ANCHOR
```

Comment:
Where does the plural come from?
I only know of a singular `.. note::` admonition as documented at https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions.

Reply:
Also, have you considered supporting all the admonitions from the list?

Reply:
reverted note(s) .. saw bad example, did not check source

unsure what you mean

Reply:
> unsure what you mean

This (from the link above):
> Directive Types
"attention", "caution", "danger", "error", "hint", "important", "note", "tip", "warning", "admonition"

You could do `_RST_NOTES = re.compile(r".. (attention|caution|danger|error|hint|important|note|tip|warning|admonition)::")` to cover all the admonitiona types, not just "note".

Reply:
decided to go back to just dealing with note, remove rest... we really dont use any of those in the plugin docs, will revise if/when we start doing so.